### PR TITLE
Cherrypick convert2rhel 3.1

### PR DESCRIPTION
**Pull request by Lennonka**
Cherry-pick of rewritten "Converting hosts to RHEL using Convert2RHEL" for Satellite 6.11
Convert2RHEL was remade into a chapter in _Managing Hosts_ instead of a standalone guide -- see #1627 (3.3+) 

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; planned orcharhino 6.4 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

**Comment by Lennonka** [comment]
@adamlazik1 @maximiliankolb Can you pls ack this?

**Review by adamlazik1 — APPROVED** [review]

File: guides/common/assembly_converting-a-host-to-rhel.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ifdef::context[:parent-context: {context}]
```
Comment (maximiliankolb) [inline guides/common/assembly_converting-a-host-to-rhel.adoc:1]: It looks like you're not overwriting `{context}`, so lines 1+2 and 6-8 are IMHO not necessary.

**Review by maximiliankolb — COMMENTED** [review]
One small suggestion, rest LGTM.

**Comment by Lennonka** [comment]
Cherry-picked:
- fa911b402..9c9e5226a  3.2 -> 3.2

**Pull request merged**